src/test/java/PlayASongTests.java

```diff
@@ -0,0 +1,61 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlayASongTests extends BaseTest {
+
+    String email = "holostenco.yuliya@gmail.com";
+    String searchPhrase = "Waiting On A Train";
+
+    @Test
+    public void playSong() throws InterruptedException {
+
+        login();
+        search();
+        playSelectedSong();
+        validatePlayOfTheSong();
+
+
+    }
+
+    private void validatePlayOfTheSong() {
+        WebElement imgOfAPlayingSong = driver.findElement(By.xpath("//img[@alt='Sound bars']"));
+        Assert.assertTrue(imgOfAPlayingSong.isDisplayed());
+    }
+
+    private void playSelectedSong() {
+        Actions actions = new Actions(driver);
+        WebElement selectedSong = driver.findElement(By.xpath("//ul/article[1]/span[2]/span[1]"));
```

Comment:
You can use a locator like this `//section[@data-testid='song-excerpts']//span[@class='details']`

![image](https://user-images.githubusercontent.com/104761482/198562566-57c6fb23-ae06-4d82-9078-51b5798d54ed.png)